tools/version/src/jarTest/java/org/apache/polaris/version/TestPolarisVersion.java

```diff
@@ -57,13 +57,23 @@ public class TestPolarisVersion {
   @Order(1)
   public void versionAvailable() {
     soft.assertThat(polarisVersionString()).isEqualTo(System.getProperty("polarisVersion"));
-    soft.assertThat(isReleaseBuild()).isFalse();
-    soft.assertThat(getBuildReleasedVersion()).isEmpty();
-    soft.assertThat(getBuildTimestamp()).isEmpty();
-    soft.assertThat(getBuildGitHead()).isEmpty();
-    soft.assertThat(getBuildGitTag()).isEmpty();
-    soft.assertThat(getBuildSystem()).isEmpty();
-    soft.assertThat(getBuildJavaVersion()).isEmpty();
+    if (isReleaseBuild()) {
+      soft.assertThat(isReleaseBuild()).isTrue();
+      soft.assertThat(getBuildReleasedVersion()).isNotEmpty();
+      soft.assertThat(getBuildTimestamp()).isNotEmpty();
```

Comment:
proposal: `soft.assertThat(getBuildTimestamp().isEmpty()).isEqualTo(!isReleaseBuild())` etc... then we do not need the big `if`